backends/ebpf/tests/ptf/test.py

```diff
@@ -60,8 +60,8 @@ class SimpleTunnelingPSATest(P4EbpfTest):
     def runTest(self):
         pkt = Ether(dst="11:11:11:11:11:11") / testutils.simple_ip_only_packet(ip_dst="192.168.1.1")
 
-        exp_pkt = Ether(dst="11:11:11:11:11:11") / MPLS(label=20, cos=5, s=1, ttl=64) / testutils.simple_ip_only_packet(
-            ip_dst="192.168.1.1")
+        exp_pkt = Ether(dst="11:11:11:11:11:11") / MPLS(
+            label=20, cos=5, s=1, ttl=64) / testutils.simple_ip_only_packet(ip_dst="192.168.1.1")
```

Comment:
This one is the most readable due to the header name being split from its content, but I don't see a good alternative, as both

```python
        exp_pkt = Ether(dst="11:11:11:11:11:11") / \
            MPLS(label=20, cos=5, s=1, ttl=64) / \
            testutils.simple_ip_only_packet(ip_dst="192.168.1.1")
```

and

```python
        exp_pkt = (Ether(dst="11:11:11:11:11:11") /
            MPLS(label=20, cos=5, s=1, ttl=64) /
            testutils.simple_ip_only_packet(ip_dst="192.168.1.1"))
```

look somewhat weird. For me, the last one might be acceptable and so is the one you have chosen. I'd probably avoid the `\` version due to weird combination with `/`.